### DISCUSSION TITLE
Where can I find this sweet PostgreSQL Grafana dashboard ?

### DISCUSSION THREAD

**Discussion by caleno** [netdata/netdata#15964] [Q&A] [answered]
Hi,

I'd really like to get my hand on this dashboard showing in the netdata-grafana-datasource-plugin repo.

![193311991-a6d167ab-b845-49b7-817c-976b780e427e](https://github.com/netdata/netdata/assets/5146433/a70254d4-35cc-4124-bc0a-1e314883cbdf)

[netdata-grafana-datasource-plugin](https://github.com/netdata/netdata-grafana-datasource-plugin)

**Comment by caleno** [comment]
@hugovalente-pm  Hi, might you know where I can find that dasboard?

**Reply by andrewm4894** [reply]
I think maybe @shyamvalsan did you make that one?

**Reply by shyamvalsan** [reply]
Yes, let me go hunt it down 🏃

**Reply by shyamvalsan** [reply, marked as answer]
Here's the [exported json](https://github.com/netdata/community/blob/main/dashboards/PostgreSQL-Grafana-Netdata.json) for this dashboard, hope this helps @caleno

**Reply by caleno** [reply]
Thank you :pray: :star_struck: :beers:
